From: Jan Engelhardt <jengelh@inai.de>

build: always pretend wxpython is available

Relying on wxpython's presence if it is not even needed for building
is a bad idea, because that causes a build cycle:

presage (pyprompter) depends on python-wxWidgets
python-wxWidgets depends on wxWidgets
wxWidgets depends on SDL2
SDL2 depends on fcitx
fcitx depends on presage (libpresage)
---
 configure.ac |   18 ------------------
 1 file changed, 18 deletions(-)

Index: presage-0.9.1/configure.ac
===================================================================
--- presage-0.9.1.orig/configure.ac
+++ presage-0.9.1/configure.ac
@@ -343,26 +343,8 @@ dnl Checks for pyprompter
 dnl =====================
 if test "$PYTHON" != :
 then
-    AC_MSG_CHECKING(for python wx module)
-    $PYTHON -c "import wx" 2&>/dev/null
-    if test $? -eq 0;
-    then
     	have_python_wx=yes
-    else
-    	have_python_wx=no
-    fi
-    AC_MSG_RESULT($have_python_wx)
-    
-    
-    AC_MSG_CHECKING(for python wx.stc module)
-    $PYTHON -c "import wx.stc" 2&>/dev/null
-    if test $? -eq 0;
-    then
     	have_python_wx_stc=yes
-    else
-    	have_python_wx_stc=no
-    fi
-    AC_MSG_RESULT($have_python_wx_stc)
 fi
 
 if test "$PYTHON" != : -a \
